### DISCUSSION TITLE
How could I count the number of visits for each room?

### DISCUSSION THREAD

**Discussion by yukipioa** [mozilla/hubs#5455] [Hubs Cloud Questions]
I have multiple rooms for one domain in one Hubs Cloud.
How could I count the number of visits for each room?

I am currently trying to count the number of visits for each room in Google Analytics, but it only counts visits for the domain as a whole.
I would appreciate your help.

**Comment by yukipioa** [comment]
I have made the following settings in ADVANCE in the Server Setting of Hubs Cloud.

I added "https://www.googletagmanager.com/" to "Extra Content Security Policy script-src Rules". Extra Content Security Policy script-src Rules".
I added Google Analytics tracking code to Extra Homepage Header HTML, Extra Room Header HTML, Extra Scene Header HTML, and Extra Avatar Header HTML.

As a result, I was able to count the number of accesses per room when accessed from the account I always use on my PC.
Active users will see the URL they are accessing.

However, if I access the room from another account that I do not use regularly on my PC, it is not counted and the active user counts as 0.

I thought it might have something to do with the room's admin permissions, but when I access from an account with admin permissions on a different PC, it is not counted.

Is there something else I need to configure?

**Reply by matthewbcool** [reply]
Hi @yukipioa,
I think injecting into the room header should be enough. It takes 24-48 hours for Analytics to process the data for reports, can you confirm you are not getting results from the realtime report?

**Reply by trongvodang** [reply]
hi @yukipioa , i encounter this problem too, any update on this? 
Thanks
